### ISSUE TITLE
Search for sites, node name is "/Sites"

### ISSUE THREAD

**Issue by slaptot** [Alfresco/alfresco-ios-app#522]
Hi,

On an Spanish OS Alfresco installation, Sites Folder is called "Sitios" on the repository. So, "Sites" default value won't works for searching.

So in "SitesListViewController.m" we found a constant called "kSitesFolderLocation" set to "/Sites" and is not localized, because it's searching for "cm:name". This constant it's used in function "searchBarSearchButtonClicked"
 
![captura de pantalla 2015-07-14 a las 11 07 00](https://cloud.githubusercontent.com/assets/1463911/8669958/062b44a6-2a19-11e5-979d-85c3a4188b78.png)

**Comment by mikeh** [comment]
Hi Alberto

My understanding of CMIS is that it uses the `cmis:path` property rather than `cmis:name` for path queries for this very reason. Just to check, could you let me know what that property is reported as in a CMIS response for your Sitios folder please? It should look something like this:
```xml
<cmis:propertyString propertyDefinitionId="cmis:path" displayName="Path" localName="path" queryName="cmis:path">
    <cmis:value>/Sites</cmis:value>
</cmis:propertyString>
```

Thanks,
Mike

**Comment by slaptot** [comment]
Hi Mike,
CMIS response:
![cmisalfresco](https://cloud.githubusercontent.com/assets/1463911/8671267/f247385e-2a23-11e5-8c06-88b17f0e458b.png)

Response is "/Sitios" instead of "/Sites"

Thanks in advance,
Alberto

**Comment by mikeh** [comment]
Thanks Alberto

I think that's a bug in the Alfresco CMIS implementation, which I'll raise here. Could you let me know which version you're running please?

In the meantime I'll see what we can do to workaround the issue with the iOS and Android apps; it looks like we'll have to enumerate by `cmis:objectTypeId` instead.

Thanks,
Mike

**Comment by slaptot** [comment]
Hi Mike,
Alfresco version 5.0.d. Let me check, but Android app is working fine.

Thanks,
Alberto

**Comment by mikeh** [comment]
Thanks Alberto

The Android app doesn't support cross-site search, so there won't be the same issue seen with that version.

Mike

**Comment by mikeh** [comment]
Having gone through the CMIS spec just now, it seems the path segments don't need to be localised at all and repository vendors *may* choose to use `cmis:name`, as is the case with Alfresco. Therefore this doesn't seem to be a repository bug.

I'll try to get this addressed in the iOS 2.2 app.

Thanks,
Mike

**Comment by slaptot** [comment]
Thank you very much Mike. It's not a critical bug for us, just trace and update iOS code :)

Kind regards,
Alberto
